packages/material-ui/src/MobileStepper/MobileStepper.test.js

```diff
@@ -71,79 +71,23 @@ describe('<MobileStepper />', () => {
 
   it('should render the back button', () => {
     const wrapper = mount(<MobileStepper {...defaultProps} />);
-    const backButton = findOutermostIntrinsic(wrapper).childAt(0);
-    assert.strictEqual(backButton.text(), 'Back');
+    const backButton = wrapper.find('button[aria-label="back"]');
+    assert.strictEqual(backButton.exists(), true);
     assert.lengthOf(backButton.find('svg[data-mui-test="KeyboardArrowLeftIcon"]'), 1);
   });
 
   it('should render next button', () => {
     const wrapper = mount(<MobileStepper {...defaultProps} />);
-    const nextButton = findOutermostIntrinsic(wrapper).childAt(2);
-    assert.strictEqual(nextButton.childAt(0).text(), 'Next');
+    const nextButton = wrapper.find('button[aria-label="next"]');
+    assert.strictEqual(nextButton.exists(), true);
     assert.lengthOf(nextButton.find('svg[data-mui-test="KeyboardArrowRightIcon"]'), 1);
   });
 
-  it('should render backButton custom text', () => {
```

Comment:
This tests the same as `should render next button`. Both specify a `nextButton` prop.